<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamSetting</class>
 <widget class="QDialog" name="ParamSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>参数设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>触发模式：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox_triggerMode">
     <item>
      <property name="text">
       <string>硬触发</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>软触发</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>测试</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>波形长度：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox_waveLength">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>64</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>128</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>256</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>512</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget">
     <row>
      <property name="text">
       <string>触发阈值</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>程控增益</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>CH1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CH2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CH3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CH4</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>200</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>200</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>200</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>200</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>5</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>5</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>5</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>5</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>138</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_save">
        <property name="text">
         <string>保存参数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
